deployment/configenv/xml_jlibpt/SWBackupNode.cpp

```diff
@@ -57,6 +52,5 @@ unsigned SWBackupNode::add(IPropertyTree *params)
        }
        compTree->addPropTree(selector, nodeGroup);
    }
-   return rc;
 }
 }
```

Comment:
Is a failure here, the property tree not added, anything that needs to be handled?

Reply:
A exception should be throw in method which return rc